### PR TITLE
Float button

### LINKED ISSUE #18: Floating Add Button
Module: Event/Finance

Criteria

- Floating Button in Event and Finance Module
- Fixed at the right bottom corner

Usage: 
Develop as a floating button used in multiple modules to link to respective form page.

- [ ] Development
- [ ] Unit Test

### PR DESCRIPTION
**Pull request by junzai97**
Resolve #18 
The button's color will only take effect after merge with #12

### CODE REVIEW

File: src/main/webapp/app/shared/layout/floatButton/FloatButton.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Button } from 'reactstrap';
+import {} from '@fortawesome/free-solid-svg-icons';
```
Comment (kachun333) [inline src/main/webapp/app/shared/layout/floatButton/FloatButton.tsx]: please remove unused import statement

**Review by kachun333 — DISMISSED** [review]

**Comment by kachun333** [comment]
This is fine to be merged, but I have a feature request. 
Please make the FAB hide on scroll, perhaps can put it into the product backlog and revisit later?

File: src/main/webapp/app/shared/layout/floatButton/FloatButton.tsx (hunk 1 above)
Comment (junzai97) [reply src/main/webapp/app/shared/layout/floatButton/FloatButton.tsx]: Resolved in 9b52ff7a844e085f0b71863cd90d5fa5cd22fb53

**Comment by junzai97** [comment]
> This is fine to be merged, but I have a feature request.
> Please make the FAB hide on scroll, perhaps can put it into the product backlog and revisit later?

**Comment by junzai97** [comment]
> This is fine to be merged, but I have a feature request.
> Please make the FAB hide on scroll, perhaps can put it into the product backlog and revisit later?

Yeah, I will prefer to merge this pull request and revisit again as an extra feature in the future

**Review by HowardYaw — APPROVED** [review]

**Comment by HowardYaw** [comment]
@junzai97  or @kachun333 
please open an issue on the feature
can be as enhancement

**Comment by junzai97** [comment]
> @junzai97 or @kachun333
> please open an issue on the feature
> can be as enhancement

Issue opened in #29

**Review by kachun333 — APPROVED** [review]

**Pull request merged**